sisl/geom/category/_coord.py

```diff
@@ -232,3 +233,55 @@ def __eq__(self, other):
                         return False
                 return True
         return False
+
+# The following code is just to make the use of coordinate categories more convenient
+# We will create one individual category for each direction. In this way, one can access
+# them directly in the keyword category builder. E.g.: `AtomCategory(x=(1,2))` creates `AtomXYZ(x=(1,2))`
+# without needing to nest "x", as in `AtomCategory(xyz={"x": (1,2)})`.
+
+
+class CoordMeta(CategoryMeta):
+    """
+    Metaclass to give extra functionality to individual coordinates categories.
+
+    The classes generated by this metaclass can use `<` and `>` to create categories.
+    See examples.
+
+    Examples
+    ----------
+
+    If `AtomX` has been built using this metaclass:
+    >>> AtomX < 5 == AtomX((None, 5)) == AtomXYZ(x=(None, 5))
+    >>> AtomX > 5 == AtomX((5, None)) == AtomXYZ(x=(5, None))
+    """
+
+    def __lt__(self, other):
+        return self(None, other)
+
+    def __gt__(self, other):
+        return self(other, None)
+
+
+def _new(cls, *interval):
+    """
+    Will go into the __new__ method of the new coordinate classes
+    """
+    if not isinstance(interval[0], Number):
+        interval = interval[0]
+    return AtomXYZ(**{cls._coord_key: interval})
+
+# Iterate over all directions that deserve an individual class
+for key in ("x", "y", "z", "f_x", "f_y", "f_z", "a_x", "a_y", "a_z"):
+
+    # The underscores are just kept for the key that is passed to AtomXYZ,
+    # but the name of the category will have no underscore
+    name = key.replace("_", "")
+
+    # Create the class for this direction
+    new_cls = CoordMeta(f"Atom{name}", (AtomCategory, ), {"__new__": _new, "_coord_key": key})
```

Comment:
for consistency I would prefer if we do `f"Atom{name.upper()}"`

Reply:
Also thought about this, but then you would access them like `AtomXYZ.FX`, which looks uglier to me than `AtomXYZ.fx`. What do you prefer, uppercase in the class name and lowercase in the attribute or all uppercase?

Reply:
Class names uppercase, attributes lowercase.